doc/classes/Font.xml

```diff
@@ -248,6 +248,7 @@
 			</argument>
 			<description>
 				Returns the size size of a bounding box of a string, taking kerning and advance into account.
+				[b]Note:[/b]  The height returned is the font height (see also [method get_height]) and has no relation to the string.
```

Comment:
You have two spaces after "Note:", only one is needed (for consistency with the rest).

Reply:
Thanks, not sure how that snuck in. Fixed!